starcheck/calc_ccd_temps.py

```diff
@@ -292,22 +299,22 @@ def get_week_states(tstart, tstop, bs_cmds, tlm):
     :param tlm: available pitch and aacccdpt telemetry recarray from fetch
     :returns: numpy recarray of states
     """
-    cstates = Table(get_cmd_states.fetch_states(DateTime(tstart) - 30,
+
+    # Get temperature data at the end of available telemetry
```

Comment:
I don't quite get all these diffs, which should be untouched by just adding `run_start_time`.  Is this about getting the regression tests looking OK?  Isn't this going to make rebasing `run_start_time_kadi` a problem?  Or I'm just confused about the plan with the two branches.

Reply:
I don't understand this comment. 

Do you mean you don't want me to also change the definition of the initial state with temperature telemetry in this PR?  Because that's what I was asking about in our sit down and you were onboard with it then.  If not clear, I was trying to say, I originally wanted three PRS

1) with run_start_time
2) with run_start_time and change in definition of initial state with temperature
3) with run_start_time + change in definition + kadi

But once I settled on skipping to 2) with the understanding that you were OK without looking into great detail into the diffs caused by changing the definition of the initial state... as that was just your code and you thought it was the right thing.  That at least made it easy to diff the cmd_states and kadi_states version.

So these changes in this section are your changes to the definition of the initial state.

And they aren't completely about "getting the regression tests looking OK" because I don't know which "OK" you mean.  It shouldn't make rebasing `run_start_time_kadi` a problem, because the block is above the area of where the kadi change happens (but did make trying to rebase off your branch too much work).

Reply:
I also tried to explain what was happening in this PR in the PR description.

Reply:
OK, that makes sense.  I admit to not carefully reading the top description and I thought this was only what the title said (I've changed it now).